src/cek/CekMachine.js

```diff
@@ -1,4 +1,6 @@
 import { expectSome } from "@helios-lang/type-utils"
+import { BasicUplcLogger } from "@helios-lang/compiler-utils"
```

Comment:
As mentioned in my review of the related compiler-utils PR (https://github.com/HeliosLang/compiler-utils/pull/3), I think we should move `BasicUplcLogger` and `UplcLoggingI` into this package, eg. as src/cek/BasicUplcLogger.js and src/cek/UplcLoggingI.js (because `UplcLoggingI` is only a typedef, you probably need to add a dummy line with `export {}` so node knows it's a module)

Reply:
got it, making a `uplc/src/logging/` directory with this